pkg/utils/default_ocm_client.go

```diff
@@ -0,0 +1,146 @@
+package utils
```

Comment:
Is it worth to move everything ocm related to it's own package ? We will work closely with ocm when we start writing controllers, I am sure this code will be reused alot and built upon.

Reply:
we can revisit this later